Octokit/Exceptions/RateLimitExceededException.cs

```diff
@@ -78,6 +81,27 @@ public override string Message
             get { return ApiErrorMessageSafe ?? "API Rate Limit exceeded"; }
         }
 
+        /// <summary>
+        /// Calculates the time from now to wait until the next request can be
+        /// attempted.
+        /// </summary>
+        /// <returns>
+        /// A non-negative <see cref="TimeSpan"/> value. Returns
+        /// <see cref="TimeSpan.Zero"/> if the next Rate Limit window has
+        /// started and the next request can be attempted immediately.
+        /// </returns>
+        /// <remarks>
+        /// The return value is calculated using server time data from the 
+        /// response in order to provide a best-effort estimate that is 
+        /// independant from eventual inaccuracies in the client's clock.
+        /// </remarks>
+        public TimeSpan GetRetryAfterTimeSpan()
```

Comment:
Can you add a check to the existing `HandlesMissingHeaderValues` test inside `Octokit.Tests/Exceptions/RateLimitExceededExceptionTests.cs`, as well as a new test using the expected headers to ensure we're not erroring somehow when doing these checks?

Reply:
@shiftkey I added new Assertions to the existing test and added new unit tests for both the Exception and the `ApiInfoParser` class.